asv_bench/asv.conf.json

```diff
@@ -41,7 +41,10 @@
         "sqlalchemy": [],
         "scipy": [],
         "numexpr": [],
-        "pytables": [],
+        "tables": [],
```

Comment:
but the 'import name' is tables I think.

In any case, we are probably not going to be able to specify a fully correct asv.conf.json file (eg also for windows an extra entry has to be added). So I would propose to choose one case (I would say: conda on linux) and make it work for that, and then clearly explain in the docs or readme what has to be changed to use it in the other cases.